magefiles/magefile.go

```diff
@@ -31,9 +31,12 @@ import (
 	"github.com/carolynvs/magex/pkg/downloads"
 	"github.com/carolynvs/magex/pkg/gopath"
 	"github.com/carolynvs/magex/shx"
+	"github.com/carolynvs/magex/xplat"
 	"github.com/magefile/mage/mg"
+	"github.com/magefile/mage/target"
 	"github.com/pkg/errors"
 	"golang.org/x/sync/errgroup"
+	// mage:import
```

Comment:
Careful running import tools on the magefiles, that comment has meaning when placed above an import line which has now been lost. In this case since the tests package is being imported with `.`, the comment isn't needed so can you just remove this stray comment now?